pkg/networkservice/common/heal/client_test.go

```diff
@@ -180,11 +193,15 @@ func TestHealClient_EmptyInit(t *testing.T) {
 			},
 		},
 	}
+
+	cancelFunc()
+	close(eventCh)
```

Comment:
using 'defer' for such call is preferred, since it guaranties the call for all exit points.
is it a special case?

Reply:
Good idea, thanks :)